File: src/main/java/com/cuupa/classificator/services/kb/semantic/SemanticResult.java
package com.cuupa.classificator.services.kb.semantic;

import java.util.List;

public class SemanticResult {

	private String topicName;

	private List<Metadata> metaData;

	public SemanticResult(String topicName) {
		this.topicName = topicName;
	}

	public String getTopicName() {
		return topicName;
	}

	public void setMetaData(List<Metadata> metaData) {
		this.metaData = metaData;
	}

	public List<Metadata> getMetaData() {
		return metaData;
	}
}

File: src/main/java/com/cuupa/classificator/knowledgebase/resultobjects/SemanticResult.kt
package com.cuupa.classificator.knowledgebase.resultobjects

data class SemanticResult(val topicName: String, var sender: String, var metaData: List<Metadata>) {

    constructor(topicName: String, metaData: MutableList<Metadata>) : this(topicName, Sender.UNKNOWN, metaData)

    constructor(topicName: String) : this(topicName, Sender.UNKNOWN, mutableListOf())
}